core/variant/variant.h

```diff
@@ -532,6 +532,7 @@ class Variant {
 	static bool is_builtin_method_static(Variant::Type p_type, const StringName &p_method);
 	static bool is_builtin_method_vararg(Variant::Type p_type, const StringName &p_method);
 	static void get_builtin_method_list(Variant::Type p_type, List<StringName> *p_list);
+	static void get_builtin_static_method_list(Variant::Type p_type, List<StringName> *p_list);
```

Comment:
I am not sure if this is really needed. You should be able to request the method list and check whether they are static. For autocompletion purposes it will probably make no difference.